redfish-core/lib/account_service.hpp

```diff
@@ -1707,6 +1784,7 @@ inline void requestAccountServiceRoutes(App& app)
                         messages::internalError(asyncResp->res);
                         return;
                     }
+
```

Comment:
empty line, these cause merge conflicts and make carrying patches harder ;)

Reply:
Should be removed now.